### PR TITLE
Correct default values

### PR DESCRIPTION
**Pull request by boutinb**
Fixes https://github.com/jasp-stats/jasp-test-release/issues/2024

### CODE REVIEW

File: inst/qml/Prophet.qml
Comment (petersen-f) [inline inst/qml/Prophet.qml:465]: I tested this and while it works, I would probably change it back to an IntegerField and instead set the Horizon option to a even value like 4.

The reason is that the Period between cutoffs defines how many periods the module shifts the training period ahead for cross validation. And if my data is in the hourly format and I set this option to 1.5 days it shifts the series ahead 36 hours which makes sense.

But for some reason prophet also allows me to set it to 1.5 hours if I have hourly data. Technically this doesn't make sense since I don't have any datapoints 1.5 hours ahead that I could fit the data to  - but it still computes.
So I would probably change it back to Integer for this release - and maybe adjust it later once I checked the code of what prophet does internally.

**Review by petersen-f — CHANGES_REQUESTED** [review]
Hey,

nice that you spotted the change point error.
I checked the other option and I would change that back since I am not sure what prophet does internally.
Cheers,
Fridtjof

File: inst/qml/Prophet.qml
Comment (boutinb) [reply inst/qml/Prophet.qml]: > I tested this and while it works, I would probably change it back to an IntegerField and instead set the Horizon option to a even value like 4.
> 
> The reason is that the Period between cutoffs defines how many periods the module shifts the training period ahead for cross validation. And if my data is in the hourly format and I set this option to 1.5 days it shifts the series ahead 36 hours which makes sense.
> 
> But for some reason prophet also allows me to set it to 1.5 hours if I have hourly data. Technically this doesn't make sense since I don't have any datapoints 1.5 hours ahead that I could fit the data to - but it still computes. So I would probably change it back to Integer for this release - and maybe adjust it later once I checked the code of what prophet does internally.

OK, then I have used the Math.floor function.

**Review by petersen-f — APPROVED** [review]
Looks good! Thank you

**Comment by petersen-f** [comment]
Ah I just realised I can't approve it since I don't have write access and this blocks merging. Could you either give me access or request review from someone else? @boutinb

**Pull request merged**
